gtsfm/data_association/point3d_initializer.py

```diff
@@ -55,6 +56,53 @@ class TriangulationParam(Enum):
     RANSAC_TOPK_BASELINES = 3  # deterministically choose hypotheses with largest estimate baseline
 
 
+class TriangulationOptions(NamedTuple):
+    """Options for triangulation solver.
+
+    Based upon COLMAP's RANSAC class:
+    Reference: https://github.com/colmap/colmap/blob/dev/src/optim/ransac.h
+
+    See the following slides for a derivation of the #req'd samples: http://www.cse.psu.edu/~rtc12/CSE486/lecture15.pdf
+
+    Args:
+        reproj_error_threshold: the maximum reprojection error allowed.
+        mode: triangulation mode, which dictates whether or not to use robust estimation.
+        min_inlier_ratio: a priori assumed minimum probability that a point is an inlier.
+        confidence: desired confidence that at least one hypothesis is outlier free.
+        dyn_num_hypotheses_multiplier: multiplication factor for dynamically computed hyptheses based on confidence.
```

Comment:
Can you elaborate what this is in the PR comments?

Also good to have an example computation in the docstring for the number of hypothesis.

Reply:
@travisdriver we could add the following math:
```python
The RANSAC module defaults to 2749 iterations, computed as 
np.log(1-0.9999) / np.log( 1 - 0.1 **2) * 3 = 2749.3
```

Reply:
I added this to the docstring